apps/dashboard/app/javascript/packs/navigator/index.js

```diff
@@ -0,0 +1 @@
+import {} from './data_table.js';
```

Comment:
Do we need this file if this is all it does?

Reply:
Unfortutanely. Yes.  I tried going directly to data_table.js and it failed.  I could only get it to work by going through index.js and import data_tables